db/kafka/scripts/topic-add.sh

```diff
@@ -34,7 +34,7 @@ fi
 eval set -- "$PARSED"
 
 # Default values.
-environment=local bootstrap=${BOOTSTRAP:-tno-kafka-broker:29094} partitions=3 replication=3 index=0
+environment=local bootstrap=${BOOTSTRAP:-kafka-broker-0:29094} partitions=3 replication=3 index=0
```

Comment:
We'll need to update with overrides for our local environments.  The original script helped developers get their environment setup.

Reply:
Thanks! I will add new script tied with production deployment script.